File: nan21.dnet.module.bp/nan21.dnet.module.bp.ui.extjs/src/main/webapp/src/ds/PaymentTermLovDs.js
 
   	 
Ext.define("net.nan21.dnet.module.bp.base.ds.model.PaymentTermLovDs" , {
	extend: 'Ext.data.Model',
	fields: [  		
		{name:"id",type:"int", useNull:true }, 		
		{name:"clientId",type:"int", useNull:true }, 		
		{name:"name",type:"string" }, 		
		{name:"active",type:"boolean" }
	]
});
	
Ext.define("net.nan21.dnet.module.bp.base.ds.param.PaymentTermLovDsParam", {
	extend: 'Ext.data.Model',
	fields: [ 
	]
}); 
 

File: nan21.dnet.module.bd/nan21.dnet.module.bd.presenter/src/main/java/net/nan21/dnet/module/bd/tx/ds/model/PaymentTermLovDs.java
/*
 * DNet eBusiness Suite
 * Copyright: 2010 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms.
 */
package net.nan21.dnet.module.bd.tx.ds.model;

import net.nan21.dnet.core.api.annotation.SortField;
import net.nan21.dnet.core.presenter.model.base.AbstractTypeLov;

import net.nan21.dnet.module.bd.tx.domain.entity.PaymentTerm;
import net.nan21.dnet.core.api.annotation.Ds;
import net.nan21.dnet.core.api.annotation.DsField;

@Ds(entity = PaymentTerm.class, jpqlWhere = " e.active = true ", sort = { @SortField(field = PaymentTermLovDs.fNAME) })
public class PaymentTermLovDs extends AbstractTypeLov<PaymentTerm> {

    public PaymentTermLovDs() {
        super();
    }

    public PaymentTermLovDs(PaymentTerm e) {
        super(e);
    }

}
